README.markdown

```diff
@@ -247,131 +449,197 @@ Returns the `basename` of a path (optionally stripping an extension). For exampl
 
 #### `bool2num`
 
-Converts a boolean to a number. Converts values:
-  * 'false', 'f', '0', 'n', and 'no' to 0.
-  * 'true', 't', '1', 'y', and 'yes' to 1.
-  Requires a single boolean or string as an input. *Type*: rvalue.
+Converts a Boolean to a number. Converts values:
+
+* `false`, 'f', '0', 'n', and 'no' to 0.
+* `true`, 't', '1', 'y', and 'yes' to 1.
+  
+  Argument: a single Boolean or string as an input.
+  
+  *Type*: rvalue.
 
 #### `bool2str`
 
-Converts a boolean to a string using optionally supplied arguments. The optional second and third arguments represent what true and false are converted to respectively. If only one argument is given, it is converted from a boolean to a string containing 'true' or 'false'.
+Converts a Boolean to a string using optionally supplied arguments. The optional second and third arguments represent what true and false are converted to respectively. If only one argument is given, it is converted from a Boolean to a string containing `true` or `false`.
 
 *Examples:*
-~~~
-bool2str(true)                    => 'true'
+
+```puppet
+bool2str(true)                    => `true`
 bool2str(true, 'yes', 'no')       => 'yes'
 bool2str(false, 't', 'f')         => 'f'
-~~~
+```
 
-Requires a single boolean as input. *Type*: rvalue.
+Arguments: Boolean.
+
+*Type*: rvalue.
 
 #### `camelcase`
 
-Converts the case of a string or all strings in an array to camel case. *Type*: rvalue.
+Converts the case of a string or all strings in an array to mixed case (both uppercase and lowercase).
+
+Arguments: TODO how do I use this function?
```

Comment:
Since camel case is (as far as I know) a technical term I would leave it in here?
Requires inputs of either array or string. Returns the same type of argument as it received but in CamelCase form.